doc/src/manual/integers-and-floating-point-numbers.md

```diff
@@ -612,6 +612,12 @@ julia> 2(x-1)^2 - 3(x-1) + 1
 3
 ```
 
+!!! warning
+    Notice that, in this case, the precedence of numeric literal coefficient is
+    still the same as that of the unary operators. Hence, the expression
+    `6/2(2+1)` leads to `1` instead of the conventional result `9`, which is
```

Comment:
@vtjnash,

Excellent, I really agree with your text. With this note (sorry I did not know that there was a note, that's why I selected warning) it should be crystal clear for everyone that the juxtaposition multiplication sometimes does not behave as the implicit multiplication in mathematics when you have a number before/after a parenthesis. If everyone agrees, then should I create a new PR with the new text and close this one?